test/e2e_node/eviction_test.go

```diff
@@ -475,6 +475,7 @@ var _ = SIGDescribe("PriorityPidEvictionOrdering", framework.WithSlow(), framewo
 
 	highPriorityClassName := f.BaseName + "-high-priority"
 	highPriority := int32(999999999)
+	processes := 30000
```

Comment:
Maybe name it `forBombProcessesCount` to make the intention clear. Alternatively, to make sure the same value is used here as for PodDisruptionConditions I would suggest to introduce a function 
`forkBombPod(name string)` calling  `pidConsumingPod(name, 30000)`. And use the function in all places.

Reply:
These tests take a long time so I'll update it once it finishes running.